lib/static/new-ui/features/suites/components/TreeViewItemSubtitle/index.tsx

```diff
@@ -16,7 +17,11 @@ interface TreeViewItemSubtitleProps {
 }
 
 export function TreeViewItemSubtitle(props: TreeViewItemSubtitleProps): ReactNode {
-    if (props.item.images?.length) {
+    if (props.item.skipReason) {
+        return <div className={styles.skipReasonContainer}>
+            <div className={styles.skipReason}>Skipped ⋅ {Parser(props.item.skipReason)}</div>
```

Comment:
` ⋅ ` - why you choose a dot as a separator?  But in IU it looks quite harmonious. Maybe you use it in some other components?

Reply:
I think it looks nice as a separator and it's not the only place we use dot. The other places are hint toast, and other variants of subtitle in tree view.